api/v1alpha1/port_types.go

```diff
@@ -33,6 +27,10 @@ type PortFilter struct {
 	// +optional
 	Description *NeutronDescription `json:"description,omitempty"`
 
+	// networkID is the ID of the attached network.
+	// +optional
+	NetworkID *UUID `json:"networkID,omitempty"`
```

Comment:
As discussed, I think we have to turn this into a `networkRef` with a dependency. This allows us to:

* Wait for a network to exist before trying to import a resource from it
* Report to the user we're waiting for a network to exist
* Refer to the network in every way supported by the Network import filter rather than just ID